build.xml

```diff
@@ -168,7 +174,18 @@
           <arg value="HEAD"/>
           <redirector output="${ssBaseDir}/test/VERSION"/>
         </exec>
+      </then>
+    </if>
+    <if>
+      <equals arg1="configTest.xml" arg2="${ssConfigFileName}"/>
+      <then>
         <copy file="${ssBaseDir}/test/search.htm_" tofile="${ssBaseDir}/test/search.html"/>
+      </then>
+    </if>
+    <if>
+      <equals arg1="configTestWorthReading.xml" arg2="${ssConfigFileName}"/>
```

Comment:
@martindholmes : Can you add a comment or something here just to explain what's happening? I'm not sure I totally understand what this is doing

Reply:
@joeytakeda I discovered that on a machine like mine where there are multiple paths to the same file, based on mount points and symlinks, using `pwd` to point to the config file when invoking the ant transformation resulted in a different path from the one Java/Ant was seeing and treating as basedir. So a straight string comparison of the full paths to the config file was failing. I think this is not an uncommon scenario for Linux users. So I've switched to just comparing the filenames. It is conceivable that another user might have their own file called "configTest.xml", but in that case, the only harmless side-effect would be the copying of search.htm_ to search.html in our test folder; there would be no impact on their own project build.

Reply:
This is all related to the need to get search.html and searchWorthReading.html out of the repo, because they change every time you run a test, and cause endless conflicts if you're working in two different branches at the same time but on completely different files that wouldn't cause merge conflicts except for those HTML files that were changing every time a test was run.

Reply:
I've added a comment (and removed a debug message I'd inadvertently left in there).

Reply:
That makes a lot of sense—thanks! I'll merge this now.